docs/learn/response_models.md

```diff
@@ -35,33 +35,38 @@ print(f"Title: {book.title}")
 print(f"Author: {book.author}")
 ```
 
+In this example:
+
+1. We define a `Book` class that inherits from Pydantic's `BaseModel`.
+2. We use the `@openai.call` decorator with the `response_model` parameter set to our `Book` class.
+3. The `recommend_book` function is decorated with both `@openai.call` and `@prompt_template`.
```

Comment:
this explanation is a little confusing.

seems like we should switch the order? i.e. (1) we write a prompt template, (2) we wrap it in the call decorator, and (3) we set the response model to `Book`?